src/ucp/wireup/select.c

```diff
@@ -65,6 +65,12 @@ typedef struct {
 } ucp_wireup_select_bw_info_t;
 
 
+typedef struct {
+    unsigned     local[UCP_MAX_RESOURCES];
```

Comment:
can use less spaces

Reply:
pls squash after fix

Reply:
actually I thought to use more (like in all other structures in this file).
leave just 1?